_data/simple-icons.json

```diff
@@ -8950,6 +8950,17 @@
             "source": "https://litecoin-foundation.org/litecoin-branding-guidelines/",
             "guidelines": "https://litecoin-foundation.org/litecoin-branding-guidelines/"
         },
+        {
+            "title": "Literal",
+            "aliases": {
+                "aka": [
+                    "Literal.club"
+                ]
+            },
```

Comment:
I think we can drop this alias.

Reply:
Okay. But they're using this handle on some of their social media.

![image](https://github.com/user-attachments/assets/19996ac8-b613-4751-9fba-331784f73594)

![image](https://github.com/user-attachments/assets/7df6fafc-26b6-427b-aae7-3d44f3850f07)

Reply:
If you look at all the places where their name is on those pages though, they use Literal. The aka aliases are for "brands that are known by different names or by their abbreviation/full name", and their handle doesn't qualify as a completely different name imo.

Reply:
Change Committed